resistor_color/resistor_color_task.py

```diff
@@ -0,0 +1,20 @@
+# Task from https://exercism.org/tracks/python/exercises/resistor-color
+
+def color_code(color):
+    switch = {
+        "black": 0,
+        "brown": 1,
+        "red": 2,
+        "orange": 3,
+        "yellow": 4,
+        "green": 5,
+        "blue": 6,
+        "violet": 7,
+        "grey": 8,
+        "white": 9
+    }
+    return switch.get(color)
+
+
+def colors():
```

Comment:
What is the purpose of this function? Should it be the constant?

Reply:
Per exercise, that should be a function. Added colors_list as a constant.
Below is a test from the website:
expected = [
    "black",
    "brown",
    "red",
    "orange",
    "yellow",
    "green",
    "blue",
    "violet",
    "grey",
    "white",
]
self.assertEqual(colors(), expected)